tests/integration/components/registries/review-actions-list/review-action/component-test.ts

```diff
@@ -0,0 +1,122 @@
+import faker from 'faker';
+import { module, test } from 'qunit';
+
+import RegistrationModel, { RegistrationReviewStates } from 'ember-osf-web/models/registration';
+import ReviewActionModel, { ReviewActionTrigger } from 'ember-osf-web/models/review-action';
+import UserModel from 'ember-osf-web/models/user';
+import formattedTimeSince from 'ember-osf-web/utils/formatted-time-since';
+
+module('Integration | Component | review-actions', hooks => {
+    setupRenderingTest(hooks);
+    setupIntl(hooks);
+    setupMirage(hooks);
+
+    hooks.beforeEach(async function(this: TestContext) {
+        this.store = this.owner.lookup('service:store');
+        const mirageRegistration = server.create('registration', 'withSingleReviewAction');
+        const registration = await this.store.findRecord('registration', mirageRegistration.id) as RegistrationModel;
+        const [reviewAction] = await registration.loadAll('reviewActions') as ReviewActionModel[];
+        const creator: UserModel = await reviewAction.creator;
+        reviewAction.setProperties({ comment: null });
+        creator.set('fullName', 'Superb Mario');
+
+        this.set('reviewAction', reviewAction);
+    });
+
+    test('Standard accept action', async function(assert) {
+        const reviewAction = this.get('reviewAction') as ReviewActionModel;
+        reviewAction.setProperties({
+            actionTrigger: ReviewActionTrigger.AcceptSubmission,
+            toState: RegistrationReviewStates.Accepted,
+        });
+        const dateString = formattedTimeSince(reviewAction.dateModified);
+        const pastTenseString = t('registries.reviewActions.triggerPastTense.accept_submission');
+
+        await render(hbs`<Registries::ReviewActionsList::ReviewAction @reviewAction={{this.reviewAction}}/>`);
+        assert.dom('[data-test-review-action-wrapper]').exists('Review action wrapper shown');
+        assert.dom('[data-test-review-action-wrapper]').containsText(
+            t('registries.reviewAction.acceptSubmission',
+                { action: pastTenseString, moderator: 'Superb Mario', date: dateString }),
+            'Review action wrapper shows proper string',
+        );
+        assert.dom('[data-test-review-action-wrapper]')
+            .doesNotContainText('contributor', 'Review action does not mention contributor');
+        assert.dom('[data-test-review-action-comment]').doesNotExist('No comment to show');
+    });
+
+    test('Embargo accept action', async function(assert) {
+        const reviewAction = this.get('reviewAction') as ReviewActionModel;
+        const embargoEndDate = faker.date.future(1, new Date(2099, 0, 0));
+        reviewAction.setProperties({
+            actionTrigger: ReviewActionTrigger.AcceptSubmission,
+            toState: RegistrationReviewStates.Embargo,
+        });
+        const dateString = formattedTimeSince(reviewAction.dateModified);
+        const embargoEndDateString = this.owner.lookup('service:intl').formatDate(embargoEndDate);
```

Comment:
you get `intl` service on the `this` context from `ember-intl/test-support` `(this: TestContext)` through `setupIntl(hooks)`
```suggestion
        const embargoEndDateString = this.intl.formatDate(embargoEndDate);
```

Reply:
Ahh, I was having trouble getting my IDE to find `this.intl` , but I wasn't binding `this: TestContext` to the test...